rsconf/component/sirepo.py

```diff
@@ -215,16 +216,13 @@ def sirepo_unit_env(self, compt=None):
         if not compt:
             compt = self
         # Only variable that is required to be in the environment
-        e = pkconfig.to_environ(
-            ["*"],
+        return self.python_service_env(
             values=PKDict(
                 (k, v) for k, v in compt.j2_ctx.items() if k in ("sirepo", "pykern")
             ),
             # local only values; exclude double under (__) which are "private" values, e.g. sirepo._run_u
-            exclude_re=r"^sirepo(?:_docker_image|_static_files|.*_vhost|.*_client_max_body|_num_api_servers|__)",
+            exclude_re=r"^sirepo(?:_docker_image|_static_files|.*_vhost|.*_client_max_body|_num_api_servers|__|__raydata)",
```

Comment:
The merge is correct. My mistake: it should be `_raydata` (one underscore).

Reply:
3685a2ea3618ee935808a36d50a01190afebbc43